sadx-input-mod/mod.cpp

```diff
@@ -15,6 +15,8 @@
 #include "input.h"
 #include "rumble.h"
 
+DataPointer(char, SoftResetByte, 0x3B0EAA0);
```

Comment:
Not a deal-breaker, but it'd be nice to avoid duplicate code. Maybe a header for variable definitions.